test/SlowTests/Issues/RavenDB-22036.cs

```diff
@@ -0,0 +1,89 @@
+using System.Linq;
+using FastTests;
+using Raven.Client;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations.Indexes;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_22036 : RavenTestBase
+{
+    public RavenDB_22036(ITestOutputHelper output) : base(output)
+    {
+    }
```

Comment:
What will happen if we have existing side-by-side reset and we reset it again?

Reply:
Added test for this case